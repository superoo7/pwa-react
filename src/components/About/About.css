.heading {
  font-size: 3rem;
}

.bio {
  line-height: 1.7;
  font-size: 1.8rem;
  color: #333333;
}
