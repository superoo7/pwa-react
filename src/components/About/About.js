import React from 'react'

import styles from './About.css'

const About = () => (
  <div>
    <h1 className={styles.heading}>Hi, I am superoo7</h1>
    <p className={styles.bio}>This is the section I talk about myself.</p>
  </div>
)

export default About
