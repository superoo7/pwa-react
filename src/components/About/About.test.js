import React from 'react'
import { shallow } from 'enzyme'
import About from './About'
import toJson from 'enzyme-to-json'


test('About component should render as expected', () => {
  const component = shallow(<About />)
  const tree = toJson(component)
  // console.log(tree)
  expect(tree).toMatchSnapshot()
})

// describe('About component', () => {
//   it('', () => {
//
//   })
// })
