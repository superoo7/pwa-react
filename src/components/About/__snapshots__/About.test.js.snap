// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render as expected 1`] = `
<div>
  <h1
    className="heading"
  >
    Hi, I am superoo7
  </h1>
  <p
    className="bio"
  >
    This is the section I talk about myself.
  </p>
</div>
`;
